cloudinit/cmd/devel/net_convert.py

```diff
@@ -4,6 +4,7 @@
 """Debug network config format conversions."""
 import argparse
 import json
+import logging
```

Comment:
If this is the only change, did we just have broken code in this module previously?

Reply:
Nice catch, it looks like the rest of this change slipped into a separate commit, I'll fix that.